### ISSUE TITLE
How to speed up the loss computation in DTSH

### ISSUE THREAD

**Issue by ssqiao** [swuxyj/DeepHash-pytorch#31]
Hi, swuxyj. Nice work for this community.
It is noted that the training loss of DTSH contains a for loop which is somewhat time-consuming. Is there any change to speed up this op? It seems that the for loop can be parallelized.

**Comment by swuxyj** [comment]
> Hi, swuxyj. Nice work for this community. It is noted that the training loss of DTSH contains a for loop which is somewhat time-consuming. Is there any change to speed up this op? It seems that the for loop can be parallelized.

可以参考其他人的实现，目前这个版本已经是我能想到的最优的方式了
